packages/typescriptlang-org/src/templates/pages/community.tsx

```diff
@@ -193,23 +182,21 @@ export const Comm: React.FC<Props> = props => {
       <div className="raised main-content-block container community">
         <h3 className="centered-highlight">Meetups</h3>
         <div className="events">
-          {Object.keys(meetupByArea)
-            .map(area => (
-              <>
-                <h4>{area}</h4>
-                <div className="callouts">
-                  {meetupByArea[area].map(meetup => (
-                    <Col className="callout">
-                      <img src={require("../../assets/community/meetup-logos/" + meetup.image)} className="icon img-square" alt={"logo of " + meetup.title} />
-                      <div>
-                        <h4 className="community-callout-headline">{meetup.title}</h4>
-                        <div className="text">{meetup.country}<br /><a rel="noopener" target="blank" href={meetup.url}>Website</a>{meetup.twitter ? <a rel="noopener" target="blank" href={meetup.twitter}>Twitter</a> : null}</div>
-                      </div>
-                    </Col>
-                  ))}
+
+          <div className="callouts">
+            {meetups.sort().map(m => m.meetup).map(meetup => (
```

Comment:
1. Is it okay that this mutates `meetups`?
2. If these are objects, what is this sorting by?
3. You can turn this into 1 `map` call